File: src/main/java/um/haberes/report/model/haberes/core/LetraDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LetraDto {

    private Long letraId;

    private Long legajoId;

    private int anho = 0;

    private int mes = 0;

    private BigDecimal neto = BigDecimal.ZERO;

    private String cadena = "";

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/LetraDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import java.math.BigDecimal

data class LetraDto(

    var letraId: Long? = null,
    var legajoId: Long? = null,
    var anho: Int = 0,
    var mes: Int = 0,
    var neto: BigDecimal = BigDecimal.ZERO,
    var cadena: String = ""

)
